omero/sysadmins/mail.txt

```diff
@@ -86,3 +86,17 @@ related property is :property:`omero.mail.bean`:
 ::
 
     $ bin/omero config set omero.mail.bean myMailImplementation
+
+OMERO.web error reporting
+-------------------------
+
+OMERO.web will email the users listed in the :property:`omero.web.admins`
```

Comment:
probably add a "property" to the end of this line after "omero.web.admins" since the reader doesn't see the `:property:` bit.